server/network.js

```diff
@@ -312,12 +311,7 @@ network.getIncidentEdges_ = function(file, gene) {
  */
 network.getComb_ = function(file, exp) {
```

Comment:
Is this function working properly now? The other things have changed a lot. I think the front-end should be sending a list of node ids rather than an array of network.Nodes.